Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/bread.yml

```diff
@@ -702,6 +702,33 @@
         Blunt: 1 # bonk
 # Tastes like France.
 
+- type: entity
+  name: cotton baguette
+  parent: FoodBreadBaguette
+  id: FoodBreadBaguetteCotton
```

Comment:
Same with introducing fiber in the solutionmanager here. I would suggest only splitting the nutriment 50-50 (i.e. 3 nutriment, 3 fiber) and keep the extra reagents like salt the same.